scripts/migrate_project_mailing_lists.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""Create mailing lists for all top-level projects
+"""
+
+import sys
+import logging
+
+from website.models import Node
+from website.app import init_app
+from website.project.mailing_list import create_list
+from website import settings
+
+logger = logging.getLogger(__name__)
+
+settings.SEARCH_ENGINE = None
+
+def migrate_node(node, dry=False):
```

Comment:
I started to write some tests for this migration script, but then realized that I didn't have a good way to imitate a Node created before my changes inside of a test. @lyndsysimon do you know of any way to do this imitation, or should I just manually set the variables to what they should be pre-changes?